### PR TITLE
BAlls Bot: the best discord bot

### PR DESCRIPTION
**Pull request by WeDoABitOfTrolling**

### CODE REVIEW

**Comment by WeDoABitOfTrolling** [comment]
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by ballsballsballsballsballsballsballsball** [comment]
haha

**Review by ballsballsballsballsballsballsballsball — CHANGES_REQUESTED** [review]
how do i show what i want changed PLEASE help me PLS

**Comment by WeDoABitOfTrolling** [comment]
:tough:

**Comment by WeDoABitOfTrolling** [comment]
WAT DO you want CHANBEDG

**Comment by ballsballsballsballsballsballsballsball** [comment]
IDK I JUST WANNA KNOW HOW I DO THE THING

**Comment by WeDoABitOfTrolling** [comment]
UHH

**Review by ballsballsballsballsballsballsballsball — APPROVED** [review]

**Comment by ballsballsballsballsballsballsballsball** [comment]
i’m so co fus

**Comment by WeDoABitOfTrolling** [comment]
smae

**Comment by WeDoABitOfTrolling** [comment]
https://haacked.com/archive/2019/06/03/suggested-changes/

File: BallsDiscordBot.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const Balls   = new BallsJS.Client();
 let token     = 'balls';
 let prefix    = '()';
 
+// haha i edit
```
Comment (ballsballsballsballsballsballsballsball) [inline BallsDiscordBot.js:6]: i want to say remove this with request change thing but HOOOW

> 

>

**Review by ballsballsballsballsballsballsballsball — CHANGES_REQUESTED** [review]

**Comment by ballsballsballsballsballsballsballsball** [comment]
I DID IT

**Comment by ballsballsballsballsballsballsballsball** [comment]
YESSSS

**Comment by WeDoABitOfTrolling** [comment]
YAY

**Comment by ballsballsballsballsballsballsballsball** [comment]
NOW REMOVE IT PLS

**Review by ballsballsballsballsballsballsballsball — APPROVED** [review]

**Pull request merged**